independent-publisher-connectors/MaintenanceRequestOxmaint/apiDefinition.swagger.json

```diff
@@ -0,0 +1,147 @@
+{
+  "swagger": "2.0",

```

Comment:
Please refer the below link and add contact and x-ms-connector-metadata property. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json